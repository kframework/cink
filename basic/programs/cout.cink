void main() {
  int a = 8;
  cout << a << endl;
}
