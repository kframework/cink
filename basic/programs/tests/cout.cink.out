8
<T>
    <in>
        ListItem(#buffer ( "" ))
        ListItem(#istream ( 0 ))
    </in>
    <out>
        ListItem(#ostream ( 1 ))
        ListItem(#buffer ( "" ))
    </out>
    <control>
        <k>
            .K
        </k>
        <return>
            noVal
        </return>
        <fstack>
            .List
        </fstack>
        <sideEffects>
            .Bag
        </sideEffects>
        <env>
            main |-> #symScalLoc(1)
        </env>
    </control>
    <genv>
        main |-> #symScalLoc(1)
    </genv>
    <store>
        #symScalLoc(1) |-> lambda (  @ int a ; (a = 8) ; ((cout << a) << (
          endl)) ; )
        #symScalLoc(2) |-> 8
    </store>
</T>
